src/Okta.Idx.Sdk.UnitTests/IdxClientShould.cs

```diff
@@ -1015,5 +1015,85 @@ public async Task CallInteractOnStartSignInWidget()
             Assert.Equal(1, mockHttpMessageHandler.CallCounts["/oauth2/v1/interact"]);
         }
 
+        [Fact]
+        public async Task CallInteractAndIntrospectOnGetIdentityProviders()
+        {
+            #region mockResponses
+            string interactResponse = @"{ ""interaction_handle"":""AcSDZw_kwcQtFUwLAgyUyBzl_OifS5Nn6IZQ_X1WXWI""}";
+            string introspectResponse = @"{
+    ""version"": ""1.0.0"",
+    ""stateHandle"": ""02WGhu-hBqJ0GVV8IjFiAU18m0fFKWv9sFNQ-E_-bP"",
+    ""expiresAt"": ""2021-05-10T18:13:24.000Z"",
+    ""intent"": ""LOGIN"",
+    ""remediation"": {
+        ""type"": ""array"",
+        ""value"": [
+            {
+                ""name"": ""redirect-idp"",
+                ""type"": ""FACEBOOK"",
+                ""idp"": {
+                    ""id"": ""test-facebook-id"",
+                    ""name"": ""Facebook IdP""
+                },
+                ""href"": ""test facebook href"",
+                ""method"": ""GET""
+            },
+            {
+                ""name"": ""redirect-idp"",
+                ""type"": ""GOOGLE"",
+                ""idp"": {
+                    ""id"": ""test-google-id"",
+                    ""name"": ""Google IdP""
+                },
+                ""href"": ""test google href"",
+                ""method"": ""GET""
+            }
+        ]
+    },
+    ""cancel"": {
+        ""rel"": [
+            ""create-form""
+        ],
+        ""name"": ""cancel"",
+        ""href"": ""https://dev-30426356.okta.com/idp/idx/cancel"",
```

Comment:
Let's mask org URLs and App IDs, I usually replace org anmes by https://foo.com/...